code/modules/jobs/job_types/roguetown/Inquisition/puritan.dm

```diff
@@ -25,9 +26,163 @@
+	outfit = /datum/outfit/job/roguetown/puritan/muscle
+	cmode_music = 'sound/music/templarofpsydonia.ogg'
+
+	category_tags = list(CTAG_PURITAN)
+
+/datum/outfit/job/roguetown/puritan/muscle/pre_equip(mob/living/carbon/human/H)
+	..()
+	shirt = /obj/item/clothing/suit/roguetown/armor/chainmail/hauberk/fluted
+	armor = /obj/item/clothing/suit/roguetown/armor/plate/blk
+	belt = /obj/item/storage/belt/rogue/leather
+	neck = /obj/item/clothing/neck/roguetown/psicross/silver
+	shoes = /obj/item/clothing/shoes/roguetown/boots/armor/blk
+	wrists = /obj/item/clothing/wrists/roguetown/bracers
+	pants = /obj/item/clothing/under/roguetown/chainlegs/blk
+	cloak = /obj/item/clothing/cloak/psydontabard
+	backr = /obj/item/storage/backpack/rogue/satchel/black
+	beltr = /obj/item/storage/belt/rogue/pouch/coins/rich
+	head = /obj/item/clothing/head/roguetown/roguehood/psydon
+	gloves = /obj/item/clothing/gloves/roguetown/chain/psydon
+	beltl = /obj/item/rogueweapon/mace/silver
+	backpack_contents = list(/obj/item/storage/keyring/puritan = 1, /obj/item/rogueweapon/huntingknife/idagger/silver)
+	if(H.mind)
+		H.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/maces, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/shields, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE) /// Swords? When you should be bashing skulls?
+		H.mind.adjust_skillrank(/datum/skill/combat/knives, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/reading, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/sewing, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 1, TRUE)
+		if(H.age == AGE_MIDDLEAGED)
+			H.mind.adjust_skillrank(/datum/skill/combat/maces, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 1, TRUE)
+		if(H.age == AGE_OLD)
+			H.mind.adjust_skillrank(/datum/skill/combat/maces, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 1, TRUE)
+			H.change_stat("endurance", 2) //Offset age 
+			H.change_stat("strength", 2) //to offset the age
+		H.change_stat("strength", 2)	///Templar stats but with perception and intelligence.
+		H.change_stat("endurance", 3) 
+		H.change_stat("constitution", 3) ///Psydonian Templar should NOT be superior to the big bossman
+		H.change_stat("perception", 2)
+		H.change_stat("intelligence", 2)
+		H.change_stat("speed", -1)
```

Comment:
+12/-1 with an additional +4 to age. If there's anything I've learned from this review it's that we probably need to nerf Inquisitor.

Reply:
Yeah I'll just give them the same shit as the Guard Captain + Keep Buff